.github/workflows/go.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: "1.20"
+        go-version: "1.21"
```

Comment:
Might as well bump it to the latest release 1.22.2?

Reply:
Yes. I'll also bump go.mod.